core/src/main/java/uk/ac/bbsrc/tgac/miso/core/service/submission/EndPoint.java

```diff
@@ -31,14 +31,14 @@
  * @author Rob Davey
  * @since 0.0.2
  */
-public interface EndPoint<T extends URI> {
+public interface EndPoint {
   /**
    * Accepts sequence Datafiles
    * 
    * @param destination
    *          String
    */
-  public void setDestination(T destination);
+  public void setDestination(URI destination);
```

Comment:
If this now has to be a URI, is there any point to subclasses, or even to this being a class? Just use a URI instead?